2019-11-28 12:56:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "CREATE DATABASE[*]  IF NOT EXISTS `KAMI_PRODS`"; se esperaba "OR, FORCE, VIEW, ALIAS, SEQUENCE, USER, TRIGGER, ROLE, SCHEMA, CONSTANT, DOMAIN, TYPE, DATATYPE, AGGREGATE, LINKED, MEMORY, CACHED, LOCAL, GLOBAL, TEMP, TEMPORARY, TABLE, SYNONYM, PRIMARY, UNIQUE, HASH, SPATIAL, INDEX"
Syntax error in SQL statement "CREATE DATABASE[*]  IF NOT EXISTS `KAMI_PRODS`"; expected "OR, FORCE, VIEW, ALIAS, SEQUENCE, USER, TRIGGER, ROLE, SCHEMA, CONSTANT, DOMAIN, TYPE, DATATYPE, AGGREGATE, LINKED, MEMORY, CACHED, LOCAL, GLOBAL, TEMP, TEMPORARY, TABLE, SYNONYM, PRIMARY, UNIQUE, HASH, SPATIAL, INDEX"; SQL statement:
CREATE DATABASE  IF NOT EXISTS `kami_prods` [42001-200]
2019-11-28 12:56:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Schema "KAMI_PRODS" no encontrado
Schema "KAMI_PRODS" not found; SQL statement:
USE `kami_prods` [90079-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.engine.Database.getSchema(Database.java:1936)
	at org.h2.command.dml.Set.update(Set.java:466)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
2019-11-28 12:56:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Tipo de dato desconocido : "TIPO"
Unknown data type: "TIPO"; SQL statement:
CREATE TABLE `personajes` (
  `id_personaje` int(100) NOT NULL AUTO_INCREMENT,
  `nombre` varchar(20) DEFAULT NULL,
  `deidad` varchar(30) DEFAULT NULL,
  `tipo` int(1) DEFAULT NULL,
  `arma` varchar(50) DEFAULT NULL,
  `ciu_nac` int(50) DEFAULT NULL,
  `proyecto` int(50) NOT NULL,
  PRIMARY KEY (`id_personaje`),
  KEY `tipo` (`tipo`),
  KEY `ciu_nac` (`ciu_nac`),
  CONSTRAINT `personajes_ibfk_1` FOREIGN KEY (`tipo`) REFERENCES `tipopersonaje` (`idtipo`),
  CONSTRAINT `personajes_ibfk_2` FOREIGN KEY (`ciu_nac`) REFERENCES `ciudades` (`idciudad`),
  CONSTRAINT `personajes_ibfk_3` FOREIGN KEY (`proyecto`) REFERENCES `proyectos` (`idproyecto`)
) [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseCreateTable(Parser.java:8379)
	at org.h2.command.Parser.parseCreate(Parser.java:6276)
	at org.h2.command.Parser.parsePrepared(Parser.java:903)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
2019-11-28 12:56:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "PERSONAJES" no encontrada
Table "PERSONAJES" not found; SQL statement:
--
-- Dumping data for table `personajes`
--

INSERT INTO `personajes` VALUES 
(1,'Zero','Tiempo y espacio',2,'Bloodborne',1, 1),
(2,'Tophox','Guerra',2,'Pena Capital',1, 1),
(3,'Caput','Sabiduría',2,'Luciferum',1, 1),
(4,'Acha','Tiempo',2,'Vane(Alfanje) y Bonnie, Low y Every (Pistolas)',1, 1),
(5,'Aga','Agua',2,'Nejibana',2, 1),
(6,'Armiche','Fuego',2,'Armas de puño',2, 1),
(7,'Tito','Formless',2,'Camaleónica',1, 1),
(8,'Knox','Muerte',3,'Knoxnomicón',1, 1),
(9,'Leo','Vida',3,'Báculo de Vida',1, 1),
(10,'Orm','Sueño',3,'Phantaso y Phobos',1, 1),
(11,'Greyman','Naturaleza',3,'Báculo de magia',3, 1),
(12,'Vento','Viento',3,'Viento doble',3, 1),
(13,'Knost','Tierra',3,'Ornstein',4, 1),
(14,'Ryu','Rayo',3,'Naginata',4, 1) [42102-200]
2019-11-28 13:01:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Nombre de columna Duplicada "ID_ITEM"
Duplicate column name "ID_ITEM"; SQL statement:
CREATE TABLE `item` (
  `id_item` int(1) NOT NULL AUTO_INCREMENT,
  `description` varchar(20) DEFAULT NULL,
  `id_item` int(1) NOT NULL AUTO_INCREMENT,
  `description` varchar(20) DEFAULT NULL,
  PRIMARY KEY (`idtipo`)
) [42121-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.setColumns(Table.java:446)
	at org.h2.table.TableBase.<init>(TableBase.java:75)
	at org.h2.table.RegularTable.<init>(RegularTable.java:126)
	at org.h2.mvstore.db.MVTable.<init>(MVTable.java:113)
	at org.h2.mvstore.db.MVTableEngine$Store.createTable(MVTableEngine.java:253)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:127)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:41)
	at org.h2.schema.Schema.createTable(Schema.java:697)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:111)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
2019-11-28 13:02:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "IDTIPO" no encontrada
Column "IDTIPO" not found; SQL statement:
CREATE TABLE `Supplier` (
  `id_supplier` int(1) NOT NULL AUTO_INCREMENT,
  `name` varchar(25) DEFAULT NULL,
  `country` varchar(25) NOT NULL,
  PRIMARY KEY (`idtipo`)
) [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.table.IndexColumn.mapColumns(IndexColumn.java:115)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:123)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.ddl.CommandWithColumns.createConstraints(CommandWithColumns.java:83)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:133)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
2019-11-28 13:09:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "CREATE TABLE `REDUCED_PRICE` (
  `ID_REDUCED_PRICE` INT(1) NOT NULL AUTO_INCREMENT,
  `START_DATE` DATE NOT NULL,
  `END_DATE` DATE NOT NULL,
)[*]"; se esperaba "identifier"
Syntax error in SQL statement "CREATE TABLE `REDUCED_PRICE` (
  `ID_REDUCED_PRICE` INT(1) NOT NULL AUTO_INCREMENT,
  `START_DATE` DATE NOT NULL,
  `END_DATE` DATE NOT NULL,
)[*]"; expected "identifier"; SQL statement:
CREATE TABLE `reduced_price` (
  `id_reduced_price` int(1) NOT NULL AUTO_INCREMENT,
  `start_date` date NOT NULL,
  `end_date` date NOT NULL,
) [42001-200]
2019-11-28 13:13:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "ITEM" ya existe
Table "ITEM" already exists; SQL statement:
CREATE TABLE `item` (
  `id_user` int(1) NOT NULL AUTO_INCREMENT,
  `name_user` varchar(20) DEFAULT NULL,
  `rol_user` int(1) NOT NULL,
 PRIMARY KEY(ID_USER)
) [42101-200]
2019-11-28 13:13:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "CREATE TABLE `ITEM` (
  `ID_ITEM` INT(1) NOT NULL AUTO_INCREMENT,
  `DESCRIPTION` VARCHAR(20) DEFAULT NULL,
  `PRICE` INT(1) NOT NULL,
  `STATE` BOOLEAN DEFAULT NULL,
  `SUPPLIERS` INT(1) NOT NULL,
  `PRICEREDUCCION` INT(1) DEFAULT NULL,
  `CREATIONDATE` INT(1) NOT NULL AUTO_INCREMENT,
  `CREATIONUSER` VARCHAR(20) DEFAULT NULL,

  PRIMARY KEY (`IDTIPO`)
  FOREIGN[*] KEY (SUPPLIERS) REFERENCES SUPPLIER(ID_SUPPLIER)
  FOREIGN KEY (PRICEREDUCCION) REFERENCES REDUCED_PRICE (ID_REDUCED_PRICE)
  FOREIGN KEY (CREATIONUSER) REFERENCES USERS(ID_USER)
)"; se esperaba "INDEX, ,, )"
Syntax error in SQL statement "CREATE TABLE `ITEM` (
  `ID_ITEM` INT(1) NOT NULL AUTO_INCREMENT,
  `DESCRIPTION` VARCHAR(20) DEFAULT NULL,
  `PRICE` INT(1) NOT NULL,
  `STATE` BOOLEAN DEFAULT NULL,
  `SUPPLIERS` INT(1) NOT NULL,
  `PRICEREDUCCION` INT(1) DEFAULT NULL,
  `CREATIONDATE` INT(1) NOT NULL AUTO_INCREMENT,
  `CREATIONUSER` VARCHAR(20) DEFAULT NULL,

  PRIMARY KEY (`IDTIPO`)
  FOREIGN[*] KEY (SUPPLIERS) REFERENCES SUPPLIER(ID_SUPPLIER)
  FOREIGN KEY (PRICEREDUCCION) REFERENCES REDUCED_PRICE (ID_REDUCED_PRICE)
  FOREIGN KEY (CREATIONUSER) REFERENCES USERS(ID_USER)
)"; expected "INDEX, ,, )"; SQL statement:
CREATE TABLE `item` (
  `id_item` int(1) NOT NULL AUTO_INCREMENT,
  `description` varchar(20) DEFAULT NULL,
  `price` int(1) NOT NULL,
  `state` boolean DEFAULT NULL,
  `suppliers` int(1) NOT NULL,
  `priceReduccion` int(1) DEFAULT NULL,
  `creationDate` int(1) NOT NULL AUTO_INCREMENT,
  `creationUser` varchar(20) DEFAULT NULL,

  PRIMARY KEY (`idtipo`)
  FOREIGN KEY (suppliers) references supplier(ID_SUPPLIER)
  FOREIGN KEY (priceReduccion) references REDUCED_PRICE (ID_REDUCED_PRICE)
  FOREIGN KEY (creationUser) references USERS(ID_USER)
) [42001-200]
2019-11-28 13:13:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "IDTIPO" no encontrada
Column "IDTIPO" not found; SQL statement:
CREATE TABLE `item` (
  `id_item` int(1) NOT NULL AUTO_INCREMENT,
  `description` varchar(20) DEFAULT NULL,
  `price` int(1) NOT NULL,
  `state` boolean DEFAULT NULL,
  `suppliers` int(1) NOT NULL,
  `priceReduccion` int(1) DEFAULT NULL,
  `creationDate` int(1) NOT NULL AUTO_INCREMENT,
  `creationUser` varchar(20) DEFAULT NULL,

  PRIMARY KEY (`idtipo`),
  FOREIGN KEY (suppliers) references supplier(ID_SUPPLIER),
  FOREIGN KEY (priceReduccion) references REDUCED_PRICE (ID_REDUCED_PRICE),
  FOREIGN KEY (creationUser) references USERS(ID_USER)
) [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.table.IndexColumn.mapColumns(IndexColumn.java:115)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:123)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.ddl.CommandWithColumns.createConstraints(CommandWithColumns.java:83)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:133)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
2019-11-28 13:22:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "ALTER TABLE	
	ITEM RENAME PRICEREDUCCION[*] TO PRICE_REDUCCION"; se esperaba "COLUMN, CONSTRAINT, TO"
Syntax error in SQL statement "ALTER TABLE	
	ITEM RENAME PRICEREDUCCION[*] TO PRICE_REDUCCION"; expected "COLUMN, CONSTRAINT, TO"; SQL statement:
ALTER TABLE	
	item RENAME pricereduccion to price_reduccion [42001-200]
2019-11-28 13:22:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "ALTER TABLE	
	ITEM RENAME PRICEREDUCCION[*] COLUMN TO PRICE_REDUCCION"; se esperaba "COLUMN, CONSTRAINT, TO"
Syntax error in SQL statement "ALTER TABLE	
	ITEM RENAME PRICEREDUCCION[*] COLUMN TO PRICE_REDUCCION"; expected "COLUMN, CONSTRAINT, TO"; SQL statement:
ALTER TABLE	
	item RENAME pricereduccion column to price_reduccion [42001-200]
2019-11-28 13:33:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Tipo de dato desconocido : 
Unknown data type: ; SQL statement:
ALTER TABLE	
	reduced_price add column REDUCED_PRICE [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:8133)
	at org.h2.command.Parser.parseAlterTable(Parser.java:7690)
	at org.h2.command.Parser.parseAlter(Parser.java:6983)
	at org.h2.command.Parser.parsePrepared(Parser.java:887)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
2019-11-28 13:33:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Tipo de dato desconocido : 
Unknown data type: ; SQL statement:
ALTER TABLE	
	reduced_price add column REDUCED_PRICE [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:8133)
	at org.h2.command.Parser.parseAlterTable(Parser.java:7690)
	at org.h2.command.Parser.parseAlter(Parser.java:6983)
	at org.h2.command.Parser.parsePrepared(Parser.java:887)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
2019-12-01 19:31:34 database: wrong user or password; user: "ROOT"
org.h2.message.DbException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:309)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:289)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:785)
	at org.h2.server.web.WebApp.test(WebApp.java:947)
	at org.h2.server.web.WebApp.process(WebApp.java:238)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 21 more
2019-12-01 19:38:38 database: wrong user or password; user: "ROOT"
org.h2.message.DbException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:309)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:289)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:785)
	at org.h2.server.web.WebApp.test(WebApp.java:947)
	at org.h2.server.web.WebApp.process(WebApp.java:238)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 21 more
2019-12-01 19:38:44 database: wrong user or password; user: "ROOT"
org.h2.message.DbException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:309)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:289)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:785)
	at org.h2.server.web.WebApp.login(WebApp.java:1008)
	at org.h2.server.web.WebApp.process(WebApp.java:223)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Nombre de usuario ó password incorrecto
Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 21 more
2019-12-01 20:39:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'prueba1' (ITEM: ""STATE"" BOOLEAN DEFAULT NULL)"
Data conversion error converting "'prueba1' (ITEM: ""STATE"" BOOLEAN DEFAULT NULL)"; SQL statement:
insert into item
values (null, 'prueba1', 1, 'prueba1',1,1, '01-01-2019','01-01-2019') [22018-200]
2019-12-01 20:39:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'prueba1' (ITEM: ""STATE"" BOOLEAN DEFAULT NULL)"
Data conversion error converting "'prueba1' (ITEM: ""STATE"" BOOLEAN DEFAULT NULL)"; SQL statement:
insert into item
values (null, 'prueba1', 1, 'prueba1',1,1, '01-01-2019','01-01-2019') [22018-200]
2019-12-01 20:42:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Tipo de dato desconocido : "DATATYPE"
Unknown data type: "DATATYPE"; SQL statement:
ALTER TABLE ITEM 
ALTER COLUMN CREATION_DATE
DATATYPE DATE [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseAlterTableAlterColumnType(Parser.java:8065)
	at org.h2.command.Parser.parseAlterTableAlter(Parser.java:7811)
	at org.h2.command.Parser.parseAlterTable(Parser.java:7698)
	at org.h2.command.Parser.parseAlter(Parser.java:6983)
	at org.h2.command.Parser.parsePrepared(Parser.java:887)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
2019-12-01 20:43:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Tipo de dato desconocido : "DATATYPE"
Unknown data type: "DATATYPE"; SQL statement:
ALTER TABLE ITEM 
ALTER COLUMN CREATION_DATE
DataType DATE [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseAlterTableAlterColumnType(Parser.java:8065)
	at org.h2.command.Parser.parseAlterTableAlter(Parser.java:7811)
	at org.h2.command.Parser.parseAlterTable(Parser.java:7698)
	at org.h2.command.Parser.parseAlter(Parser.java:6983)
	at org.h2.command.Parser.parsePrepared(Parser.java:887)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
2019-12-01 20:43:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Tipo de dato desconocido : "DATATYPE"
Unknown data type: "DATATYPE"; SQL statement:
ALTER TABLE ITEM 
ALTER COLUMN CREATION_DATE DataType DATE [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseAlterTableAlterColumnType(Parser.java:8065)
	at org.h2.command.Parser.parseAlterTableAlter(Parser.java:7811)
	at org.h2.command.Parser.parseAlterTable(Parser.java:7698)
	at org.h2.command.Parser.parseAlter(Parser.java:6983)
	at org.h2.command.Parser.parsePrepared(Parser.java:887)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
2019-12-01 20:44:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Tipo de dato desconocido : "START_DATE"
Unknown data type: "START_DATE"; SQL statement:
ALTER TABLE ITEM 
ALTER COLUMN CREATION_DATE start_date DATE [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseAlterTableAlterColumnType(Parser.java:8065)
	at org.h2.command.Parser.parseAlterTableAlter(Parser.java:7811)
	at org.h2.command.Parser.parseAlterTable(Parser.java:7698)
	at org.h2.command.Parser.parseAlter(Parser.java:6983)
	at org.h2.command.Parser.parsePrepared(Parser.java:887)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
2019-12-01 20:45:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'prueba1' (ITEM: ""STATE"" BOOLEAN DEFAULT NULL)"
Data conversion error converting "'prueba1' (ITEM: ""STATE"" BOOLEAN DEFAULT NULL)"; SQL statement:
insert into item
values (null, 'prueba1', 1, 'prueba1',1,1, '01-01-2019','prurebauser') [22018-200]
2019-12-01 20:45:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "prueba1" no encontrada
Column "prueba1" not found; SQL statement:
insert into item
values (null, "prueba1", 1, 'prueba1',1,1, '01-01-2019','prurebauser') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
2019-12-01 20:46:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "prueba1" no encontrada
Column "prueba1" not found; SQL statement:
insert into item (ID_ITEM, DESCRIPTION, PRICE, STATE, SUPPLIERS, PRICE_REDUCCION, CREATION_DATE, CREATION_USER)
values (null, "prueba1", 1, 'prueba1',1,1, '01-01-2019','prurebauser') [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Insert.prepare(Insert.java:327)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
2019-12-01 20:47:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'prueba1' (ITEM: ""STATE"" BOOLEAN DEFAULT NULL)"
Data conversion error converting "'prueba1' (ITEM: ""STATE"" BOOLEAN DEFAULT NULL)"; SQL statement:
INSERT INTO ITEM
VALUES (null, 'prueba1', 1, 'prueba1',1,1, '01-01-2019','prurebauser') [22018-200]
2019-12-01 20:48:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'prueba1' (ITEM: ""STATE"" BOOLEAN DEFAULT NULL)"
Data conversion error converting "'prueba1' (ITEM: ""STATE"" BOOLEAN DEFAULT NULL)"; SQL statement:
INSERT INTO ITEM
VALUES (1, 'prueba1', 1, 'prueba1',1,1, '01-01-2019','prurebauser') [22018-200]
2019-12-01 20:48:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'prueba2' (ITEM: ""STATE"" BOOLEAN DEFAULT NULL)"
Data conversion error converting "'prueba2' (ITEM: ""STATE"" BOOLEAN DEFAULT NULL)"; SQL statement:
INSERT INTO ITEM
VALUES (null, 'prueba1', 1, 'prueba2',1,1, '01-01-2019','prurebauser') [22018-200]
2019-12-01 20:49:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Imposible interpretar la constante "DATE" "01-01-2019"
Cannot parse "DATE" constant "01-01-2019"; SQL statement:
INSERT INTO ITEM
VALUES (null, 'prueba1', 1, '0',1,1, '01-01-2019','prurebauser') [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.IllegalArgumentException: 1-1-2019
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:173)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 19 more
2019-12-01 20:51:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Columna "CREATION_" no encontrada
Column "CREATION_" not found; SQL statement:
ALTER TABLE ITEM
ALTER COLUMN CREATION_ DATE Timestamp [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:719)
	at org.h2.command.Parser.columnIfTableExists(Parser.java:8052)
	at org.h2.command.Parser.parseAlterTableAlter(Parser.java:7712)
	at org.h2.command.Parser.parseAlterTable(Parser.java:7698)
	at org.h2.command.Parser.parseAlter(Parser.java:6983)
	at org.h2.command.Parser.parsePrepared(Parser.java:887)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
2019-12-01 20:52:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Imposible interpretar la constante "TIMESTAMP" "01-01-2019"
Cannot parse "TIMESTAMP" constant "01-01-2019"; SQL statement:
INSERT INTO ITEM
VALUES (null, 'prueba1', 1, '0',1,1, '01-01-2019','prurebauser') [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:136)
	at org.h2.value.Value.convertToTimestamp(Value.java:1169)
	at org.h2.value.Value.convertTo(Value.java:822)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.IllegalArgumentException: 1-1-2019
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:173)
	at org.h2.util.DateTimeUtils.parseTimestamp(DateTimeUtils.java:316)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:134)
	... 19 more
2019-12-01 20:53:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "INSERT INTO ITEM
VALUES (NULL, 'prueba1', 1, '0',1,1, '{ts '2019[*]-01-01 18:47:52.69'}','prurebauser')"; se esperaba "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"
Syntax error in SQL statement "INSERT INTO ITEM
VALUES (NULL, 'prueba1', 1, '0',1,1, '{ts '2019[*]-01-01 18:47:52.69'}','prurebauser')"; expected "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
INSERT INTO ITEM
VALUES (null, 'prueba1', 1, '0',1,1, '{ts '2019-01-01 18:47:52.69'}','prurebauser') [42001-200]
2019-12-01 20:53:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "INSERT INTO ITEM
VALUES (NULL, 'prueba1', 1, '0',1,1, '{ts '2019[*]-01-01 18:47:52.69'}','prurebauser')"; se esperaba "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"
Syntax error in SQL statement "INSERT INTO ITEM
VALUES (NULL, 'prueba1', 1, '0',1,1, '{ts '2019[*]-01-01 18:47:52.69'}','prurebauser')"; expected "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
INSERT INTO ITEM
VALUES (null, 'prueba1', 1, '0',1,1, '{ts '2019-01-01 18:47:52.69'}','prurebauser') [42001-200]
2019-12-01 20:53:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "INSERT INTO ITEM
VALUES (NULL, 'prueba1', 1, '0',1,1, '{ts '2019[*]-01-01 18:47:52.69'}','prurebauser')"; se esperaba "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"
Syntax error in SQL statement "INSERT INTO ITEM
VALUES (NULL, 'prueba1', 1, '0',1,1, '{ts '2019[*]-01-01 18:47:52.69'}','prurebauser')"; expected "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
INSERT INTO ITEM
VALUES (null, 'prueba1', 1, '0',1,1, '{ts '2019-01-01 18:47:52.69'}','prurebauser') [42001-200]
2019-12-01 21:06:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "INSERT INTO REDUCED_PRICE
VALUES (NULL, TO_DATE('2015-12-12 08:00 A.M.', TO_DATE('2015-12-12 08:00 A.M.', 20)[*]"; se esperaba "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"
Syntax error in SQL statement "INSERT INTO REDUCED_PRICE
VALUES (NULL, TO_DATE('2015-12-12 08:00 A.M.', TO_DATE('2015-12-12 08:00 A.M.', 20)[*]"; expected "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into REDUCED_PRICE
VALUES (null, TO_DATE('2015-12-12 08:00 A.M.', TO_DATE('2015-12-12 08:00 A.M.', 20) [42001-200]
2019-12-01 21:09:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "INSERT INTO REDUCED_PRICE
VALUES (NULL, 
TO_DATE('2015-12-12 08:00 A.M.', 
TO_DATE('2015-12-12 08:00 A.M.', 
20)[*]"; se esperaba "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"
Syntax error in SQL statement "INSERT INTO REDUCED_PRICE
VALUES (NULL, 
TO_DATE('2015-12-12 08:00 A.M.', 
TO_DATE('2015-12-12 08:00 A.M.', 
20)[*]"; expected "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into REDUCED_PRICE
VALUES (null, 
TO_DATE('2015-12-12 08:00 A.M.', 
TO_DATE('2015-12-12 08:00 A.M.', 
20) [42001-200]
2019-12-01 21:10:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "INSERT INTO REDUCED_PRICE
VALUES (
TO_DATE('2015-12-12 08:00 A.M.', 
TO_DATE('2015-12-12 08:00 A.M.', 
20)[*]"; se esperaba "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"
Syntax error in SQL statement "INSERT INTO REDUCED_PRICE
VALUES (
TO_DATE('2015-12-12 08:00 A.M.', 
TO_DATE('2015-12-12 08:00 A.M.', 
20)[*]"; expected "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into REDUCED_PRICE
VALUES (
TO_DATE('2015-12-12 08:00 A.M.', 
TO_DATE('2015-12-12 08:00 A.M.', 
20) [42001-200]
2019-12-01 21:11:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "INSERT INTO REDUCED_PRICE
VALUES (1,
TO_DATE('2015-12-12 08:00 A.M.', 
TO_DATE('2015-12-12 08:00 A.M.', 
20)[*]"; se esperaba "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"
Syntax error in SQL statement "INSERT INTO REDUCED_PRICE
VALUES (1,
TO_DATE('2015-12-12 08:00 A.M.', 
TO_DATE('2015-12-12 08:00 A.M.', 
20)[*]"; expected "[, ::, AT, FORMAT, *, /, %, +, -, ||, ~, !~, NOT, LIKE, ILIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into REDUCED_PRICE
VALUES (1,
TO_DATE('2015-12-12 08:00 A.M.', 
TO_DATE('2015-12-12 08:00 A.M.', 
20) [42001-200]
2019-12-01 21:13:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Conversión de datos fallida, convirtiendo "'Admin' (USERS: ""ROL_USER"" INT NOT NULL)"
Data conversion error converting "'Admin' (USERS: ""ROL_USER"" INT NOT NULL)"; SQL statement:
INSERT INTO USERS
VALUES (1, 'Admin', 'Admin') [22018-200]
2019-12-01 21:13:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "SERS[*]"; se esperaba "SET, SAVEPOINT, SCRIPT, SHUTDOWN, SHOW"
Syntax error in SQL statement "SERS[*]"; expected "SET, SAVEPOINT, SCRIPT, SHUTDOWN, SHOW"; SQL statement:
SERS [42001-200]
2019-12-01 21:14:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "ITEMS" no encontrada
Table "ITEMS" not found; SQL statement:
SELECT * FROM ITEMS [42102-200]
2019-12-01 21:14:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Imposible interpretar la constante "TIMESTAMP" "04-01-2019"
Cannot parse "TIMESTAMP" constant "04-01-2019"; SQL statement:
INSERT INTO "PUBLIC"."ITEM"("ID_ITEM","DESCRIPTION","PRICE","STATE","SUPPLIERS","PRICE_REDUCCION","CREATION_DATE","CREATION_USER")VALUES(?,?,?,?,?,?,?,?) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:136)
	at org.h2.value.Value.convertToTimestamp(Value.java:1169)
	at org.h2.value.Value.convertTo(Value.java:822)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:324)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:3015)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1138)
	at org.h2.server.web.WebApp.process(WebApp.java:247)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.IllegalArgumentException: 4-1-2019
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:173)
	at org.h2.util.DateTimeUtils.parseTimestamp(DateTimeUtils.java:316)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:134)
	... 20 more
2019-12-01 21:14:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Imposible interpretar la constante "TIMESTAMP" "04-01-2019"
Cannot parse "TIMESTAMP" constant "04-01-2019"; SQL statement:
INSERT INTO "PUBLIC"."ITEM"("ID_ITEM","DESCRIPTION","PRICE","STATE","SUPPLIERS","PRICE_REDUCCION","CREATION_DATE","CREATION_USER")VALUES(?,?,?,?,?,?,?,?) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:136)
	at org.h2.value.Value.convertToTimestamp(Value.java:1169)
	at org.h2.value.Value.convertTo(Value.java:822)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:324)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:3015)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1138)
	at org.h2.server.web.WebApp.process(WebApp.java:247)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.IllegalArgumentException: 4-1-2019
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:173)
	at org.h2.util.DateTimeUtils.parseTimestamp(DateTimeUtils.java:316)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:134)
	... 20 more
2019-12-01 21:15:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Imposible interpretar la constante "TIMESTAMP" "'2001-02-30 12:00:00"
Cannot parse "TIMESTAMP" constant "'2001-02-30 12:00:00"; SQL statement:
INSERT INTO "PUBLIC"."ITEM"("ID_ITEM","DESCRIPTION","PRICE","STATE","SUPPLIERS","PRICE_REDUCCION","CREATION_DATE","CREATION_USER")VALUES(?,?,?,?,?,?,?,?) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:136)
	at org.h2.value.Value.convertToTimestamp(Value.java:1169)
	at org.h2.value.Value.convertTo(Value.java:822)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:324)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:3015)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1138)
	at org.h2.server.web.WebApp.process(WebApp.java:247)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NumberFormatException: For input string: "'2001"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:569)
	at java.lang.Integer.parseInt(Integer.java:615)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:169)
	at org.h2.util.DateTimeUtils.parseTimestamp(DateTimeUtils.java:316)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:134)
	... 20 more
2019-12-01 21:15:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Imposible interpretar la constante "TIMESTAMP" "'2001-02-30 12:00:00"
Cannot parse "TIMESTAMP" constant "'2001-02-30 12:00:00"; SQL statement:
INSERT INTO "PUBLIC"."ITEM"("ID_ITEM","DESCRIPTION","PRICE","STATE","SUPPLIERS","PRICE_REDUCCION","CREATION_DATE","CREATION_USER")VALUES(?,?,?,?,?,?,?,?) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:136)
	at org.h2.value.Value.convertToTimestamp(Value.java:1169)
	at org.h2.value.Value.convertTo(Value.java:822)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:324)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:3015)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1138)
	at org.h2.server.web.WebApp.process(WebApp.java:247)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NumberFormatException: For input string: "'2001"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:569)
	at java.lang.Integer.parseInt(Integer.java:615)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:169)
	at org.h2.util.DateTimeUtils.parseTimestamp(DateTimeUtils.java:316)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:134)
	... 20 more
2019-12-01 21:19:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Imposible interpretar la constante "TIMESTAMP" "2007-January-01"
Cannot parse "TIMESTAMP" constant "2007-January-01"; SQL statement:
INSERT INTO "PUBLIC"."ITEM"("ID_ITEM","DESCRIPTION","PRICE","STATE","SUPPLIERS","PRICE_REDUCCION","CREATION_DATE","CREATION_USER")VALUES(?,?,?,?,?,?,?,?) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:136)
	at org.h2.value.Value.convertToTimestamp(Value.java:1169)
	at org.h2.value.Value.convertTo(Value.java:822)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:324)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:3015)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1138)
	at org.h2.server.web.WebApp.process(WebApp.java:247)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NumberFormatException: January
	at org.h2.util.StringUtils.parseUInt31(StringUtils.java:1020)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:170)
	at org.h2.util.DateTimeUtils.parseTimestamp(DateTimeUtils.java:316)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:134)
	... 20 more
2019-12-01 21:19:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Imposible interpretar la constante "TIMESTAMP" "2007-January-01"
Cannot parse "TIMESTAMP" constant "2007-January-01"; SQL statement:
INSERT INTO "PUBLIC"."ITEM"("ID_ITEM","DESCRIPTION","PRICE","STATE","SUPPLIERS","PRICE_REDUCCION","CREATION_DATE","CREATION_USER")VALUES(?,?,?,?,?,?,?,?) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:136)
	at org.h2.value.Value.convertToTimestamp(Value.java:1169)
	at org.h2.value.Value.convertTo(Value.java:822)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:324)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:3015)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1138)
	at org.h2.server.web.WebApp.process(WebApp.java:247)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NumberFormatException: January
	at org.h2.util.StringUtils.parseUInt31(StringUtils.java:1020)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:170)
	at org.h2.util.DateTimeUtils.parseTimestamp(DateTimeUtils.java:316)
	at org.h2.value.ValueTimestamp.parse(ValueTimestamp.java:134)
	... 20 more
